op-chain-ops/genesis/setters.go

```diff
@@ -29,23 +29,22 @@ func SetProxies(hh *hardhat.Hardhat, db vm.StateDB) error {
 	if err != nil {
 		return err
 	}
-	proxyAdmin, err := hh.GetDeployment("ProxyAdmin")
-	if err != nil {
-		return err
-	}
+	proxyAdminSlot := predeploys.ProxyAdminAddr.Hash()
```

Comment:
Nit: `proxyAdminSlotValue` might be more legible. Generally I assume the slot is the key and not the value which made this confusing to read at first.

Reply:
Nit on the nit:
`proxyAdminSlotValue` feels overload. Maybe:

- `proxyAdminValue`
- `proxyAdminPadded`